common/supplemental/languageGroup.xml

```diff
@@ -9,32 +9,32 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<languageGroup parent="awd">aca ame aox apu arw avo bae brg bvv bwi cab cbb ccc cjo cni cot cox cpc crb cuj gae gqn guc ign inp kav kgm kpc mcb mht mmh mpd mpw mzx not pab pbg pib pio plu pnk rgr sar tae ter tnq trn unk wap wau xir yaw ycn yvt</languageGroup>
 		<languageGroup parent="azc">chl cok com crn cup hch hop lui mfy mnr nah ncx nhi npl ntp ood opt pao par pia ser shh stp tac tar tbu tcu tep thh tla tub twr ute var xaw xgf xpo yaq</languageGroup>
 		<languageGroup parent="bad">bjo kuw lna lnl mnh nue tor</languageGroup>
 		<languageGroup parent="bai">bbj bko byv fmp jgo nla nnh nnz nwe xmg ybb</languageGroup>
 		<languageGroup parent="bat">lsl lt ltg lv olt prg sgs svx sxl xcu xgl xsv xzm</languageGroup>
-		<languageGroup parent="ber">auj cnu gha gho gnc grr jbe jbn kab mzb nxm oua rif sds shi shy siz sjs swn taq tez thv thz tia tjo tmh tzm zen zgh</languageGroup>
-		<languageGroup parent="bnt">abb agh asa auh axk baf bag bas bau bbg bbi bbm bbu bbx bdp beb bem beq bez bip biw biz bja bkf bkh bkj bkw bli blv bmb bmg bml bmv bmw bng bni bnm bnx boh bou boy bpj bqm bqu bqz brf bri brl bsi bss buf bui bum buu bvb bvx bwc bwg bwl bwt bww bwz bxg bxp byi bzm cce ccl cgg chw cjk coh cuh cwa cwb cwe dav dbm dbo dde dez dhm dhs dig dii diu diz dma dmx dne doe dov dua dug dzn ebo ebu ekm eko eto ewo fip flr fom fwe gev gey gmx gog gua guz gwb gwe gwr gyi haq hav hay hba heh hem hij hka hke hng hol hoo hz ifm ikz ilb isn iyx jgk jit jjr jku jmc job kam kbj kbs kck kcu kcv kcz kdc kde kdn keb ked kel kg khu khx khy ki kiv kiz kki kkj kkw kmb kme kmw kny koo koq kqn ksb ksf ksv ktf ktu kty kuj kwm kwn kws kwu kwy kzn kzo kzy lag lai lch ldi lea lel leo lfa lg lgm lie lik lke llb lli ln lnb lnz lol lon loo loq loz lse lsm lu lua lue luj lum lun luy lwa lyn mbo mck mcp mcx mdp mdq mdt mdw mer mfo mfu mgg mgh mgq mgr mgs mgv mgw mgy mgz mhb mhd mhm mho mhw mjh mkk mkw mlk mma mmu mmz mny mow mpa mvw mwe mwn mws mwz mxc mxg mxo myc mye myx mzd nba nd ndc ndh ndj ndk ndq ndw ng ngc ngl ngp ngv nih nim njd njx njy nka nkn nkt nkv nle nlj nmg nmq nnb nne nnq noq now nql nr nra nse nso nsx nte ntk nto nui nuj nvo nxi nxo ny nyc nye nyf nyg nyj nyk nym nyn nyo nyr nyu nyy nzb nzd nzu okc old olu ozm pae pbr pem phm pic piw pkb pnd pof poy ppp puu rag rim rn rnd rng rnw rof rub ruc ruf rui rw rwk rwm sak sbk sbm sbp sbs sdj seg seh sgm shc shq shr sie skt slx smx sn sng snq soc soo sop sox soz ss ssc st sub suj suk suw sw swb swj swk sxb sxe syi szv tap tbt tck tek tga thk tke tlj tll tmv tn tny tog toh toi ts tsc tsv ttf ttj tum tvs tvu twl two twx tyi tyx ukh umb vau ve vid vif vin vmk vmr vmw vum vun wbh wbi wdd wlc wmw wni won wum wun xdo xh xku xkv xma xmb xmc xog xsq yaf yao yat yav yax yel yey yko ymg ymk yns yom zaj zdj zga zin ziw zmf zms zmw zmx zu</languageGroup>
+		<languageGroup parent="ber">aao aeb arq ary auj ayl gha gnc jbn kab mt nxm siz sqr swn taq tez thv thz tmh xaa zen zgh</languageGroup>
```

Comment:
Half of these languages are different. This seems like a surprisingly large change...?